android/app/src/main/java/org/fossasia/openevent/utils/AuthUtil.java

```diff
@@ -34,23 +36,45 @@
 
     private static Authenticator authenticator;
 
-    public static void signUpUser(Context context, String email, String password, ProgressBar progressBar) {
+    public static void signUpUser(String email, String password, MutableLiveData<Integer> signUpResponse) {
```

Comment:
The idea is to deconstruct AuthUtil class, move this whole method inside the signup viewmodel. I mean there is no point of hoisting a single use method in a different class